soc/nxp/kinetis/k6x/Kconfig.defconfig

```diff
@@ -19,4 +19,10 @@ config GPIO
 config SPI
 	default n if SOC_MK64F12
 
+# The flash option register (FOPT) boot options
+# 1111 1001 - Set NMI pin/interrupts and EzPort reset default to disabled
+config KINETIS_FLASH_CONFIG_FOPT
+	default 0xF9
```

Comment:
The RM suggests that we need to set bits 1 and 2:
![image](https://github.com/user-attachments/assets/8f262644-a5fa-4322-9bff-4a093b5a9ce3)

Also, why are we setting the reserved bits to 0xF?

Should this value be 0x6?

Reply:
_>The RM suggests we need to set bits 1 and 2_

RM does not suggests, it tells the default values  - erased flash.

EezPort & NMI is very problematic and should be disabled for applications. 
If have a time, read about the pain:
- https://community.nxp.com/t5/Kinetis-Microcontrollers/Kinetis-K02-reset-issue/m-p/1067426
- https://github.com/ARMmbed/mbed-os/issues/569
- https://community.nxp.com/t5/Kinetis-Microcontrollers/NMI-pin-floating/m-p/739809
- https://mcuoneclipse.com/2016/12/22/disabling-ezport-on-nxp-kinetis-to-solve-power-on-issues/
- https://mcuoneclipse.com/2020/06/09/disabling-nmi-non-maskable-interrupt-pin/

>Also, why are we setting the reserved bits to 0xF?

It's reserved, so it set to the default value 0xF (erased flash).

>Should this value be 0x6?

It should be 0x9 (1001).

Reply:
Sorry, I was reading the RM wrong- we do need to clear the `EZPORT_DIS` bit, not set it. Looking at this again I agree it should be 0x9, and I follow the reasoning for setting the remaining bits to 0xF